pkg/controllers/user/logging/utils/splunk.go

```diff
@@ -24,15 +24,15 @@ type splunkTestWrap struct {
 func (w *splunkTestWrap) TestReachable(dial dialer.Dialer, includeSendTestLog bool) error {
 	url, err := url.Parse(w.Endpoint)
 	if err != nil {
-		return errors.Wrapf(err, "parse url %s failed", url)
+		return errors.Wrapf(err, "couldn't parse url %s", w.Endpoint)
 	}
 
 	isTLS := url.Scheme == "https"
 	var tlsConfig *tls.Config
 	if isTLS {
 		tlsConfig, err = buildTLSConfig(w.Certificate, w.ClientCert, w.ClientKey, w.ClientKeyPass, "", url.Hostname(), w.SSLVerify)
 		if err != nil {
-			return errors.Wrap(err, "build tls config failed")
+			return err
```

Comment:
Please explain a little why it doesn’t require error wrapping here.

Reply:
because error is from buildTLSConfig, we already wrap error inside buildTLSConfig function